rust/tools/authorization-logic/.gitignore

```diff
@@ -0,0 +1,11 @@
+target/
+src/parsing/antlr_gen/*interp
```

Comment:
I have not yet tried to run this code.

Files from a directory outside the source tree can be included using the `include!` macro. See e.g. https://github.com/project-oak/oak/blob/main/oak_abi/src/proto/mod.rs where the results of generating code from proto files are included.

Reply:
OK thanks. I have tried getting this to work [in this WIP commit in another branch](https://github.com/aferr/raksha/commit/5be551e7709be8af26d243c227b370a12263741e), but I get errors when trying to compile the generated rust code. The errors are about annotations in the generated code, for example:
```
error: an inner attribute is not permitted in this context
 --> src/parsing/antlr_gen/../../../target/antlr_gen/authlogiclexer.rs:2:1
  |
2 | #![allow(dead_code)]
```

It looks like [other people](https://github.com/google/flatbuffers/issues/6261) have run into similar issues with annotations in generated code. It seems like the only fix people on the internet have found is to change the generated code (which is going to be hard in this case).

Do you see another way to fix it? Or is it fine to leave the directory where code gets generated here since the actual generated code is still not tracked by the repository?

Reply:
I think it is OK to leave as is for now. It was just a suggestion, as having a mix of tracked and untracked code files in the same directory tree can be confusing.

Reply:
OK sonds good. I do agree that it would be better so I will at least track it with [this issue](https://github.com/google-research/raksha/issues/14)